const processError = async (error) => console.error(error);

export { processError };
